src/collections/_documentation/apm/distributed-tracing.md

```diff
@@ -0,0 +1,344 @@
+
+```javascript
+const transaction = Sentry.getCurrentHub().startSpan({ op: "task",  transaction: item.getTransaction() })
+  // processItem may create more spans internally (see next example)
+  processItem(item).then(() => {
+    transaction.finish()
+  })
+```
+
+**Adding Additional Spans to the Transaction**
+
+The next example is called in the `processItem` function from the code snippet above. Our SDK can determine if there is a current open `transaction` and add all newly created [spans]({%- link _documentation/apm/apm-glossary.md -%}#span) as child operations to the `transaction`. Keep in mind; each individual span also needs to be finished; otherwise, it will not show up in the `transaction`.
+
+```javascript
+function processItem(item, transaction) {
+  const span = transaction.child({ op: "http", description: "GET /" })
+
+  return new Promise((resolve, reject) => {
+    http.get(`/items/${item.id}`, (response) => {
+      response.on('data', () => {});
+      response.on('end', () => {
+        span.setTag("http.status_code", response.statusCode);
+        span.setData("http.foobarsessionid", getFoobarSessionid(response));
+        span.finish();
+        resolve(response);
+      });
+    });
+  });
+}
+```
+
+### Node.js
+
+To access our APM features, you need to use a beta release.
+
+```bash
+$ npm install @sentry/node
+$ npm install @sentry/apm
+```
+
+**Sending Traces**
+
+To send any traces, set the `tracesSampleRate` to a nonzero value. The following configuration will capture 10% of all your transactions:
+
+```javascript
+const Sentry = require("@sentry/node");
+const Apm = require("@sentry/apm"); // This is required since it patches functions on the hub
+
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  tracesSampleRate: 0.1
+});
+```
+
+**Automating Traces**
+
+It’s possible to add tracing to all popular frameworks; however, we provide pre-written handlers only for Express.js.
+
+```javascript
+const Sentry = require("@sentry/node");
+const express = require("express");
+const app = express();
+
+// RequestHandler creates a separate execution-context using domains, so that every transaction/span/breadcrumb has it's own Hub to be attached to
+app.use(Sentry.Handlers.requestHandler());
+// TracingHandler creates a tracing for every incoming request
+app.use(Sentry.Handlers.tracingHandler());
+
+// the rest of your app
+
+app.use(Sentry.Handlers.errorHandler());
+app.listen(3000);
+```
+
+The following operations have [span]({%- link _documentation/apm/apm-glossary.md -%}#span) capabilities within a [transaction]({%- link _documentation/apm/apm-glossary.md -%}#transaction):
+
+- HTTP requests made with `request` or `get` calls using native `http` and `https` modules
+- Middlewares (Express.js only)
+
+To enable them:
+
+```bash
+$ npm install @sentry/integrations@5.8.0-beta.0
```

Comment:
@HazAT @kamilogorek Is this version accurate? v5.13.2 is already out http://npm.im/@sentry/integrations

Reply:
It's definitely out of date. It shouldn't use a hardcoded version. There's a tag for that: `@sentry/integrations@{% sdk_version sentry.javascript.browser %}`